File: demo_sleep是否消耗cpu/demo_sleep是否消耗cpu/ViewController.m
//
//  ViewController.m
//  demo_sleep是否消耗cpu
//
//  Created by Lwgfangz on 14-7-25.
//  Copyright (c) 2014年 aiDriving.com. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}
- (IBAction)action:(id)sender
{
    NSLog(@"开始睡觉");
    sleep(10);
    NSLog(@"睡觉玩了");
}

@end

File: demo_swift/demo_swift/ViewController.swift
//
//  ViewController.swift
//  demo_swift
//
//  Created by Lwgfangz on 14-7-17.
//  Copyright (c) 2014年 aiDriving.com. All rights reserved.
//

import UIKit

class ViewController: UIViewController
{
                            
    override func viewDidLoad()
    {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning()
    {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


    @IBOutlet var textField : UITextField
    @IBAction func action(sender : AnyObject)
    {
        textField.text = "4";
        //UIView *view = []
    }
    @IBOutlet var button_act : UIButton
}

